prime-router/build.gradle.kts

```diff
@@ -562,6 +563,18 @@ flyway {
     url = dbUrl
     user = dbUser
     password = dbPassword
+    cleanDisabled = false
+    mixed = true
+    pluginConfiguration = mutableMapOf(
+        "postgresqlTransactionalLock" to "false"
+    )
```

Comment:
This is supposedly how it should be fixed, but this does not work. I don't think these values are pass through from the plugin down into the actual tool running the migration. Leaving for now so we can control it here if it starts working again.

Reply:
@jeremy-page Do not resolve these type of comments, so we can all see them.